website/src/pages/forum.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import Layout from '@theme/Layout';
 import Head from '@docusaurus/Head';
-import DiscourseFeed from '@site/src/components/discourse';
+import { DiscourseFeed } from '@site/src/components/discourse';
 
 function Events() {
```

Comment:
Is this a copy-paste hangover too? And does it matter?

Reply:
Since we added a second `DiscourseHelpFeed` component (which i added directly in the original component file), I had to change the way the components were exported. It was originally set to export 1 default component: `export default`, but since the file now has 2 components I changed to named exports `export const <variable-name>`. This allows the imports to be destructured with `import { DiscourseFeed, DiscourseHelpFeed }` as needed